frontend/device/sdl/device.lua

```diff
@@ -6,6 +6,42 @@ local logger = require("logger")
 local function yes() return true end
 local function no() return false end
 
+-- xdg-open is used on most linux systems
+local function hasXdgOpen()
+    local std_out = io.popen("xdg-open --version 2>/dev/null")
+    local version = nil
+    if std_out ~= nil then
+        version = std_out:read()
+        std_out:close()
+    end
+    return version ~= nil
+end
+
+-- open is the macOS counterpart
```

Comment:
And `start` on Windows. Not relevant to the code, but I was curious and looked it up.